devtools-ci-index.yaml

```diff
@@ -1228,9 +1228,10 @@
     name: '{ci_project}-{git_repo}-prcheck-{test_url}'
     triggers:
         - github
-        - githubprb:
+        - github-pull-request:
```

Comment:
i dont see this trigger defined anywhere ?

Reply:
I thought it is just something like "default trigger" for Jenkins as that article says: https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request 
And I got my inspiration from another job already defined in this yaml: 
https://github.com/Katka92/almighty-jobs/blob/b1676229b7152379aa9f5583ccc6854b46fb4341/devtools-ci-index.yaml#L248

Reply:
I thought it is just something like "default trigger" for Jenkins as that article says: https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request
And I got my inspiration from another job already defined in this yaml:
https://github.com/Katka92/almighty-jobs/blob/b1676229b7152379aa9f5583ccc6854b46fb4341/devtools-ci-index.yaml#L248